stripe.go

```diff
@@ -539,6 +550,91 @@ func SetAppInfo(info *AppInfo) {
 	initUserAgent()
 }
 
+// SetBackend sets the backend used in the binding.
+func SetBackend(backend SupportedBackend, b Backend) {
+	switch backend {
+	case APIBackend:
+		backends.API = b
+	case UploadsBackend:
+		backends.Uploads = b
+	}
+}
+
+// SetHTTPClient overrides the default HTTP client.
+// This is useful if you're running in a Google AppEngine environment
+// where the http.DefaultClient is not available.
+func SetHTTPClient(client *http.Client) {
+	httpClient = client
+}
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
+	return &v
+}
+
+// StringValue returns the value of the string pointer passed in or
+// "" if the pointer is nil.
+func StringValue(v *string) string {
```

Comment:
I wonder if it hurts discovery that this is not grouped with the other ones. But I do see the value of alphabetical ordering.

Reply:
Yeah, I thought about that as well :/

I think the best argument for keeping it sorted like this is that if you ever use any kind of Go-related tooling (say view the package in Godoc or use vim-go to do autocompletion), they'll all end up sorted like we see here because those tools will make no distinction about where a symbol is located in the source file.

IMO our best bet will be to keep it as it is here, but then introduce an explicit section of documentation in the source/Godoc that talks just about these helpers and lists each of them explicitly.